ui/ui-components/components/linkButton.tsx

```diff
@@ -0,0 +1,33 @@
+import Link from "next/link";
```

Comment:
This is a minor nitpick, and against the established pattern, but I typically try to `PascalCase` my component filenames since React requires components to be `FirstLetterUpperCase` during use.

Reply:
That's probably what we should do - story added to get them all at once - https://www.pivotaltracker.com/story/show/180158840